examples/mnist_template.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import print_function
+import argparse
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torchvision import datasets, transforms
+from torch.optim.lr_scheduler import StepLR
+
+###### Hydra Block ######
+from omegaconf import MISSING
+import hydra
+from hydra.core.config_store import ConfigStore
+from dataclasses import dataclass
+
+# config schema imports
+from config.torch.optim import AdadeltaConf
+from config.torch.optim.lr_scheduler import StepLRConf
+
+@dataclass
+class CommonArgparseArgs:
+    stuff: int = 1
+
+@dataclass
+class ExportedArgparseArgs:
+    epochs: int = 14
+    batch_size: int = 64
+    test_batch_size: int = 1000
+    no_cuda: bool = False
+    save_model: bool = False
+    dry_run: bool = False
+    log_interval: int = 10
+    seed: int = 1
+
+@dataclass
+class MNISTStepLRConf(StepLRConf):
+    step_size: int = 1 
```

Comment:
cleaner to just register StepLRConf(step_size=1) directly instead of creating a class for this.